### PR TITLE
Expand L0_perf_resnet for HTTP

### PR DESCRIPTION
**Pull request by CoderHam**
- remove dynamic batching

### CODE REVIEW

File: qa/L0_perf_resnet/run_test.sh [hunk 1]
```diff
@@ -43,95 +42,77 @@ export CUDA_VISIBLE_DEVICES=0
...
+if (( $SERVER_PID == 0 )); then
+    echo -e "\n***\n*** Failed to start $SERVER\n***"
+    cat $SERVER_LOG
+    exit 1
+fi
+
+
+set +e
+
+PERF_CLIENT_PROTOCOL_ARGS="-i grpc -u localhost:8001" &&
```
Comment (tanmayv25) [inline qa/L0_perf_resnet/run_test.sh:70]: You can directly pass the -i  $PERF_CLIENT_PROTOCOL as the cli option, the perf_client will automaitcally select the appropriate url and port number.

File: qa/L0_perf_resnet/test.sh [hunk 2]
```diff
@@ -137,73 +141,30 @@ $CAFFE2PLAN -h -b ${STATIC_BATCH} \
...
+        FRAMEWORK=$(echo ${MODEL_NAME} | cut -d '_' -f 3,4)
+        MODEL_NAME=${MODEL_NAME} \
+                MODEL_FRAMEWORK=${FRAMEWORK} \
+                MODEL_PATH="$REPO/${MODEL_NAME}" \
+                STATIC_BATCH=${STATIC_BATCH} \
+                PERF_CLIENT_PROTOCOL=${PROTOCOL} \
+                INSTANCE_CNT=${INSTANCE_CNT} \
+                bash -x run_test.sh
+    done
+done
```
Comment (tanmayv25) [inline qa/L0_perf_resnet/test.sh:170]: Add a new line

File: qa/L0_perf_resnet/run_test.sh [hunk 3]
```diff
@@ -43,95 +42,77 @@ export CUDA_VISIBLE_DEVICES=0
...
-            fi
-
-            rm -fr models && mkdir -p models && \
-                cp -r $MODEL_PATH models/. && \
-                (cd models/$MODEL_NAME && \
-                        sed -i "s/^max_batch_size:.*/max_batch_size: ${MAX_BATCH}/" config.pbtxt && \
-                        echo "instance_group [ { count: ${INSTANCE_CNT} }]" >> config.pbtxt)
-            if (( $DYNAMIC_BATCH > 1 )); then
-                (cd models/$MODEL_NAME && \
-                        echo "dynamic_batching { preferred_batch_size: [ ${DYNAMIC_BATCH} ] }" >> config.pbtxt)
```
Comment (tanmayv25) [inline qa/L0_perf_resnet/run_test.sh:74]: Why are we removing the dynamic batching benchmarks from L0_perf_resnet?

File: qa/L0_perf_resnet/test.sh [hunk 4]
```diff
@@ -89,29 +90,33 @@ $CAFFE2PLAN -h -b ${STATIC_BATCH} \
...
-              bash -x run_test.sh
+    for PROTOCOL in $PROTOCOLS; do
+        REPO=`pwd`/tensorrt_models && [ "$MODEL_NAME" != "$TRT_MODEL_NAME" ] && \
+            REPO=$REPODIR/perf_model_store
+        FRAMEWORK=$(echo ${MODEL_NAME} | cut -d '_' -f 3)
+        MODEL_NAME=${MODEL_NAME} \
+                MODEL_FRAMEWORK=${FRAMEWORK} \
+                MODEL_PATH="$REPO/${MODEL_NAME}" \
+                STATIC_BATCH=${STATIC_BATCH} \
+                PERF_CLIENT_PROTOCOL=${PROTOCOL} \
```
Comment (tanmayv25) [inline qa/L0_perf_resnet/test.sh:101]: PERF_CLIENT_PROTOCOL -> PROTOCOL

File: qa/L0_perf_resnet/run_test.sh (hunk 3 above)
Comment (CoderHam) [reply qa/L0_perf_resnet/run_test.sh:74]: It didn't test exactly what we wanted to test. After discussing with @deadeyegoodwin we decided to remove it from this test and add another test later on that tests dynamic batching performance more accurately.

File: qa/L0_perf_resnet/test.sh (hunk 4 above)
Comment (CoderHam) [reply qa/L0_perf_resnet/test.sh:101]: I kept the name as PERF_CLIENT_PROTOCOL to keep it homogeneous with the rest of the benchmark tests.

File: qa/L0_perf_resnet/run_test.sh (hunk 3 above)
Comment (tanmayv25) [reply qa/L0_perf_resnet/run_test.sh:74]: Can you elaborate on how it doesn't test what we are interested in? Do we want to test a larger set of preferred_batch_size to ascertain the performance of our dynamic_batcher?

File: qa/L0_perf_resnet/run_test.sh [hunk 5]
```diff
@@ -67,7 +67,7 @@ fi
 
 set +e
 
-PERF_CLIENT_PROTOCOL_ARGS="-i grpc -u localhost:8001" &&
+PERF_CLIENT_PROTOCOL_ARGS="-i grpc" &&
```
Comment (tanmayv25) [inline qa/L0_perf_resnet/run_test.sh:70]: I meant you can drop PERF_CLIENT_PROTOCOL_ARGS altogether and invoke perf_client like this:

`$PERF_CLIENT -v -i ${PERF_CLIENT_PROTOCOL} -m $MODEL_NAME -p5000 -b${STATIC_BATCH}`

You can make the same change in L0_perf_nomodel to maintain the homogeneity.

File: qa/L0_perf_resnet/run_test.sh (hunk 3 above)
Comment (CoderHam) [reply qa/L0_perf_resnet/run_test.sh:74]: I think David can better answer that question. We can discuss this on slack further.

**Pull request merged**